src/pages/TransactionHistory/components/FilterGroup.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  CheckboxGroup,
+  Collapse,
+  Icon,
+  Radio,
+  RadioGroup,
+  useColorModeValue
+} from '@chakra-ui/react'
+import { ReactChild, useState } from 'react'
+import { Control, useController } from 'react-hook-form'
+import { IoIosArrowDown, IoIosArrowUp } from 'react-icons/io'
+import { Text } from 'components/Text'
+
+type Option = [string, string, ReactChild?]
+
+export const FilterGroup = ({
+  title,
+  options,
+  allowMultipleOptions = false,
+  name,
+  control
+}: {
+  title: string
+  options: Option[]
+  allowMultipleOptions?: boolean
+  name: string
+  control: Control
+}) => {
+  const [isOpen, setOpen] = useState(false)
+  const toggleDayRange = () => setOpen(open => !open)
+  const {
+    field: { onChange, value }
+  } = useController({ control, name })
+  const GroupComponent = allowMultipleOptions ? CheckboxGroup : RadioGroup
+  const InputComponent = allowMultipleOptions ? Checkbox : Radio
+  return (
+    <Box px={2}>
+      <Button
+        justifyContent='space-between'
+        alignItems='center'
+        variant='ghost'
+        fontWeight='400'
```

Comment:
Same here, can leave it at the same weight.

Reply:
done